filament/backend/include/private/backend/OpenGLPlatform.h

```diff
@@ -107,6 +107,11 @@ class OpenGLPlatform : public DefaultPlatform {
     // called once before a SAMPLER_EXTERNAL texture is destroyed.
     virtual void destroyExternalImage(void* texture) noexcept {}
 
+    // allows platforms to convert the given image object into a new type (e.g. HardwareBuffer => EGLImage)
+    // by default, performs no conversion.
+    virtual AcquiredImage createAcquiredImage(void* hwbuffer, backend::StreamCallback cb, void* userData) noexcept {
```

Comment:
I prefer we implement this in the cpp. Also, is this even needed? Could it be pure virtual?

Reply:
Done, also simplified the signature and renamed it to `transformAcquiredImage`.